### PR TITLE
Update docs about debugging native code

### PR DESCRIPTION
**Pull request by tuliocll**
Hi guys, I just update the docs adding info about debugging native code with a way to use the IDE debuggers on running RN applications.
Inglish is not my native language, so if there is something wrong appoint me 😅

### CODE REVIEW

**Comment by iamstarkdev** [comment]
Thanks @tuliocll, this solved my problem!!! <3

**Review by Simek — COMMENTED** [review]
Hello @tuliocll, thank you for providing an additional information! 🙏 

Left a few minor suggestion related to the formatting, but overall this looks good.

Can you also update at least the latest stable release versioned doc?
* https://github.com/facebook/react-native-website/tree/main/website/versioned_docs/version-0.71

**Comment by tuliocll** [comment]
> Hello @tuliocll, thank you for providing an additional information! 🙏
> 
> Left a few minor suggestion related to the formatting, but overall this looks good.
> 
> Can you also update at least the latest stable release versioned doc?
> 
> * https://github.com/facebook/react-native-website/tree/main/website/versioned_docs/version-0.71

Done!

**Review by Simek — APPROVED** [review]
LGTM, thank you for the backport! 🚀

File: docs/debugging.md [hunk 1]
```diff
@@ -200,3 +200,13 @@ Alternatively, select "Dev Settings" from the Developer Menu, then update the "D
...
 
 When working with native code, such as when writing native modules, you can launch the app from Android Studio or Xcode and take advantage of the native debugging features (setting up breakpoints, etc.) as you would in case of building a standard native app.
+
+Another option is to run your application using the React Native CLI and attach the native debugger of the native IDE (Android Studio or Xcode) to the process.
+
+### Android Studio
+
+On Android Studio you can do this by going on the "Run" option on the menu bar, clicking on "Attach to Process..." and selecting the running React Native app.
+
+### XCode
```
Comment (Simek) [inline docs/debugging.md:210]:
```suggestion
### Xcode
```

File: website/versioned_docs/version-0.71/debugging.md [hunk 2]
```diff
@@ -200,3 +200,13 @@ Alternatively, select "Dev Settings" from the Developer Menu, then update the "D
...
 
 When working with native code, such as when writing native modules, you can launch the app from Android Studio or Xcode and take advantage of the native debugging features (setting up breakpoints, etc.) as you would in case of building a standard native app.
+
+Another option is to run your application using the React Native CLI and attach the native debugger of the native IDE (Android Studio or Xcode) to the process.
+
+### Android Studio
+
+On Android Studio you can do this by going on the "Run" option on the menu bar, clicking on "Attach to Process..." and selecting the running React Native app.
+
+### XCode
```
Comment (Simek) [inline website/versioned_docs/version-0.71/debugging.md:210]:
```suggestion
### Xcode
```

**Pull request merged**